test/views/email_view.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Cog.EmailViewTest do
+  use Cog.ConnCase, async: true
+
+  test "reset_url generates the proper url" do
+    Application.put_env(:cog, :password_reset_base_url, "http://example.com:4000/reset-password")
```

Comment:
Ah, I think I see what your saying now. `password_reset_base_url` isn't a url for Cog. It's a url for flywheel. Technically it could be a url for any webui for Cog but...

Let me explain how the password reset bits work. All of the view elements live in flywheel, Cog just functions as the api for resets. Flywheel will `post` to `/v1/password_reset` with an email_address in the body. If the email is a valid user email cog will add a record to the password_resets table in the db. It will then send an email with a url generated by appending the password reset id to `password_reset_base_url`. `password_reset_base_url` is a view in flywheel. So when the user clicks the link they are actually taken to flywheel where they can enter their new password. Then flywheel makes the request back to cog to actually do the reset.

Does that make sense? Forgive me if I sound elementary. For some reason this was hard to explain.